app/routes/_layout.roles.tsx

```diff
@@ -39,15 +43,48 @@ export const meta: MetaFunction = () => [
 ];
 
 export async function loader({ request }: LoaderFunctionArgs) {
-  return new RolesApi(SherlockConfiguration)
-    .apiRolesV3Get({}, handleIAP(request))
-    .then((roles) => roles.sort(roleSorter), errorResponseThrower);
+  const [roles, selfUser] = await Promise.all([
+    new RolesApi(SherlockConfiguration)
+      .apiRolesV3Get({}, handleIAP(request))
+      .then((roles) => roles.sort(roleSorter), errorResponseThrower),
+    new UsersApi(SherlockConfiguration)
+      .apiUsersV3SelectorGet({ selector: "self" }, handleIAP(request))
+      .catch(errorResponseThrower),
+  ]);
+
+  const superAdminRoles = roles.filter((r) => r.grantsSherlockSuperAdmin);
+
+  const selfUserInSuperAdminGroups = await Promise.all(
```

Comment:
this is what determines if selfUser is a super admin - happy for cleaner suggestions!